### PR TITLE
Add Link for "Operator White Paper".

### PR DESCRIPTION
**Pull request by Shubham82**
Add Link for Operator White Paper in Operator pattern docs.

Fixes: #28621

### CODE REVIEW

**Comment by Shubham82** [comment]
Hi @daminisatya @kbarnard10
Could you please take a look.

File: content/en/docs/concepts/extend-kubernetes/operator.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Operator.
 ## {{% heading "whatsnext" %}}
 
 
+* Learn about [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md)
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/operator.md:127]:
```suggestion
* Read the {{< glossary_tooltip text="CNCF" term_id="cncf" >}] [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
[Preview](https://deploy-preview-28625--kubernetes-io-main-staging.netlify.app/docs/concepts/extend-kubernetes/operator/#what-s-next)

**Comment by Shubham82** [comment]
hi @sftim 
PTAL!

Thanks

File: content/en/docs/concepts/extend-kubernetes/operator.md [hunk 2]
```diff
@@ -124,6 +124,7 @@ Operator.
 ## {{% heading "whatsnext" %}}
 
 
+* Read the {{< glossary_tooltip text="CNCF" term_id="cncf" >}] [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/operator.md:127]: Oops
```suggestion
* Read the {{< glossary_tooltip text="CNCF" term_id="cncf" >}} [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```

**Comment by Shubham82** [comment]
@sftim PTAL

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**